src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -248,6 +248,10 @@ module.exports = class LevelBlock {
     return this.isEmpty || this.isEntity;
   }
 
+  static isWalkable(blockType) {
+    return new LevelBlock(blockType).isWalkable;
+  }
```

Comment:
@joshlory I'm not sure how I feel about this approach to exposing block-level concepts; I'd love to hear what you think about it

Reply:
Not a huge fan... can we use `block.isWalkable` below?

Ideally we'd stop referencing `blockType` at all in the View, but that will take a bit of cleanup.

Reply:
Agreed that ideally we'd do that, but as it current stands there is nowhere in the call stack of this method that is actually creating a LevelBlock for us to call `isWalkable` on